### PR TITLE
docs: Added guide for distributed minio on Windows

### PR DESCRIPTION
**Pull request by koolhead17**
Added guide for distributed minio on Windows with Docker.

## Description
Added docs as requested from community for installation steps via docker on setting up minio distributed.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by deekoder** [comment]
Where does this guide reside? Is it a recipe ?

**Review by deekoder — COMMENTED** [review]
If this is not a recipe, remove numbers next to headings. Otherwise LGTM (assuming testing is complete).

**Comment by harshavardhana** [comment]
> If this is not a recipe, remove numbers next to headings. Otherwise LGTM (assuming testing is complete).

This is not a recipe and it should be part of the docker quick start guide (perhaps under windows) section. what do you think @deekoder ?

File: docs/docker/distributed-minio-docker-windows.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Distributed Minio Docker Guide for Windows [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/minio/minio?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+## 1. Prerequisites
+
+Install Install Docker Toolbox for Windows from [here](http://www.docker.com/products/docker-toolbox)
```
Comment (krisis) [inline docs/docker/distributed-minio-docker-windows.md]: typo: s/Install Install/Install

File: docs/docker/distributed-minio-docker-windows.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+```
+curl https://raw.githubusercontent.com/minio/minio/master/docs/docker/docker-compose.yml > docker-compose.yml
+```
+### Start Minio distributed cluster.
+
+```
+docker-compose.exe up
+```
+Minio browser will be simply accessible at one of the port 9001-9004.
```
Comment (krisis) [inline docs/docker/distributed-minio-docker-windows.md]: s/port/ports

**Review by krisis — CHANGES_REQUESTED** [review]

File: docs/docker/distributed-minio-docker-windows.md (hunk 1 above)
Comment (deekoder) [reply docs/docker/distributed-minio-docker-windows.md]: ack.

File: docs/docker/distributed-minio-docker-windows.md (hunk 2 above)
Comment (deekoder) [reply docs/docker/distributed-minio-docker-windows.md]: ack.

**Comment by deekoder** [comment]
@harshavardhana - It should be in the docker quickstart guide.

**Comment by koolhead17** [comment]
@deekoder @harshavardhana If you can suggest me on where it should go, i will do specific amendments and sent updated PR. 

Also @deekoder https://github.com/minio/minio/tree/master/docs/docker  the docker quick start guide has numbering in it, are you suggesting to remove those as well?

**Comment by deekoder** [comment]
@koolhead17 - We need to be consistent thats all. Minio QS has no #s. Where as others do. Let's make them all exactly similar.

**Comment by koolhead17** [comment]
@deekoder sure. It is following the style guideline with existing docker quick start guide which has numbering in it.

**Comment by deekoder** [comment]
ok LGTM.

**Pull request merged**